helm/flink-operator/templates/flink-operator.yaml

```diff
@@ -118,6 +124,9 @@ metadata:
   labels:
     {{- include "flink-operator.labels" . | nindent 4 }}
 data:
+  flink-conf.yaml: |+
```

Comment:
I am suggesting to use the `PrometheusReporter` as default. And maybe we also need to add the prometheus related annotations so that it could integrate with the metrics system easily.

https://stackoverflow.com/questions/53365191/monitor-custom-kubernetes-pod-metrics-using-prometheus

Reply:
Make sense to me.